docs/feature_rgb_matrix.md

```diff
@@ -742,6 +742,27 @@ void rgb_matrix_indicators_advanced_user(uint8_t led_min, uint8_t led_max) {
 }
 ```
 
+Layer indicator with only configured keys:
+```c
+void rgb_matrix_indicators_user(void) {
```

Comment:
```suggestion
void rgb_matrix_indicators_advanced_user(uint8_t led_min, uint8_t led_max) {
```
Was this intended to be an advanced example? As `led_min` and `led_max` aren't declared otherwise.